shell/packages/sandstorm-ui-powerbox/powerbox-client.js

```diff
@@ -151,12 +92,7 @@ SandstormPowerboxRequest = class SandstormPowerboxRequest {
             this._requestInfo.source.postMessage({
               rpcId: this._requestInfo.rpcId,
               token: apiToken,
-              descriptor: {
-                tags: [
-                  { id: "15831515641881813735" }, // UiView
-                ],
-                quality: "acceptable",
-              },
+              descriptor: "EAZQAQEAABEBF1EEAQH_5-Jn6pjXtNsAAAA",
```

Comment:
Please leave a comment about what this blob represents, to make it clearer to the next person to read this code.  Otherwise, they have to pull out a urlsafe base64 decoder, pycapnp or node-capnp, and the Sandstorm schemas to understand/verify the intent of this line.